test/rekt/features/channel/control_plane.go

```diff
@@ -84,19 +85,20 @@ func ControlPlaneChannel(channelName string) *feature.Feature {
 
 	f.Stable("Spec Requirements").
 		Must("Each channel CRD MUST contain an array of subscribers: spec.subscribers",
-			channelAllowsSubscribers)
+			channelAllowsSubscribersAndStatus)
```

Comment:
we should also change the description of the Must? We're testing two things at once here?

Reply:
Joined.